Error when SVG fill attribute is empty

```svg
<svg viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <path d="M512.002 512.002m-491.988 0a491.988 491.988 0 1 0 983.976 0 491.988 491.988 0 1 0-983.976 0Z" fill=""></path>
</svg>
```

```shell
nanoemoji --color_format glyf_colr_1 $(find ../noto-emoji/svg -name 'emoji_u270d.svg')
Wrote  /Users/yisi/works/nanoemoji/build/config.toml
Generating build.ninja
ninja -C /Users/yisi/works/nanoemoji/build
ninja: Entering directory `/Users/yisi/works/nanoemoji/build'
[4/4] /Users/yisi/works/nanoemoji/tutorial-env/bin/python3 -m nanoemoji.write_font  --config config.toml -v 1 @Font.ttf.rsp
FAILED: Font.ttf
/Users/yisi/works/nanoemoji/tutorial-env/bin/python3 -m nanoemoji.write_font  --config config.toml -v 1 @Font.ttf.rsp
I0130 16:05:08.929934 4679280064 write_font.py:423] 0 codepoints require blanks
I0130 16:05:08.930158 4679280064 color_glyph.py:320]  ColorGlyph for /Users/yisi/works/nanoemoji/build/picosvg/regular/emoji_u270d.svg ((9997,))
Traceback (most recent call last):
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/write_font.py", line 501, in <module>
    app.run(main)
  File "/Users/yisi/works/nanoemoji/tutorial-env/lib/python3.8/site-packages/absl/app.py", line 303, in run
    _run_main(main, args)
  File "/Users/yisi/works/nanoemoji/tutorial-env/lib/python3.8/site-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/write_font.py", line 495, in main
    ufo, ttfont = _generate_color_font(font_config, inputs)
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/write_font.py", line 440, in _generate_color_font
    color_glyphs = [
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/write_font.py", line 441, in <listcomp>
    ColorGlyph.create(
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/color_glyph.py", line 332, in create
    painted_layers = tuple(
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/color_glyph.py", line 269, in _painted_layers
    for (paint, normalized), paths in groupby(
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/color_glyph.py", line 271, in <lambda>
    key=lambda s: _in_glyph_reuse_key(
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/color_glyph.py", line 249, in _in_glyph_reuse_key
    _paint(debug_hint, upem, picosvg, shape),
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/color_glyph.py", line 240, in _paint
    return PaintSolid(color=Color.fromstring(shape.fill, alpha=shape.opacity))
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/colors.py", line 217, in fromstring
    raise ValueError(f"invalid or unsupported color string: {s!r}")
ValueError: invalid or unsupported color string: ''
ninja: build stopped: subcommand failed.
Traceback (most recent call last):
  File "/Users/yisi/works/nanoemoji/tutorial-env/bin/nanoemoji", line 11, in <module>
    load_entry_point('nanoemoji', 'console_scripts', 'nanoemoji')()
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/nanoemoji.py", line 375, in main
    app.run(_run)
  File "/Users/yisi/works/nanoemoji/tutorial-env/lib/python3.8/site-packages/absl/app.py", line 303, in run
    _run_main(main, args)
  File "/Users/yisi/works/nanoemoji/tutorial-env/lib/python3.8/site-packages/absl/app.py", line 251, in _run_main
    sys.exit(main(argv))
  File "/Users/yisi/works/nanoemoji/src/nanoemoji/nanoemoji.py", line 366, in _run
    subprocess.run(ninja_cmd, check=True)
  File "/usr/local/Cellar/python@3.8/3.8.5/Frameworks/Python.framework/Versions/3.8/lib/python3.8/subprocess.py", line 512, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['ninja', '-C', '/Users/yisi/works/nanoemoji/build']' returned non-zero exit status 1.
```

--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@ setup(
         "importlib_resources>=3.3.0; python_version < '3.9'",
         "lxml>=4.0",
         "ninja>=1.10.0.post1",
-        "picosvg>=0.11.0",
+        "picosvg>=0.11.1",
         "pillow>=7.2.0",
         "regex>=2020.4.4",
         "toml>=0.10.1",

